### PR TITLE
[SRN-93] Auto populate carrier fields

### PR DESCRIPTION
**Pull request by thangnguyen-novobi**

### CODE REVIEW

File: novobi_addons/sirena_fedex/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        if self.env.user.company_id.country_id.code != 'US' or not fedex:
+            return result
+        else:
+            self.shipping_account_id = fedex
+
+            if self.partner_id.address_classification == 'RESIDENTIAL':
+                for dc_id in fedex.delivery_carrier_ids.ids:
+                    delivery_carrier = self.env['delivery.carrier'].search([('id', '=', dc_id)])
+                    if delivery_carrier.fedex_service_type == 'GROUND_HOME_DELIVERY':
+                        self.delivery_carrier_id = delivery_carrier
```
Comment (tiennguyen-novobi) [inline novobi_addons/sirena_fedex/models/stock_picking.py:21]:
```
delivery_carrier_id = fedex.delivery_carrier_ids.filtered(lambda r: r.fedex_service_type == 'GROUND_HOME_DELIVERY')
self.delivery_carrier_id = delivery_carrier_id and delivery_carrier_id[0]
```

File: novobi_addons/sirena_fedex/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            else:
+                for dc_id in fedex.delivery_carrier_ids.ids:
+                    delivery_carrier = self.env['delivery.carrier'].search([('id', '=', dc_id)])
+                    if delivery_carrier.fedex_service_type == 'FEDEX_GROUND':
+                        self.delivery_carrier_id = delivery_carrier
+
```
Comment (tiennguyen-novobi) [inline novobi_addons/sirena_fedex/models/stock_picking.py:27]: Similar to above

**Review by tiennguyen-novobi — CHANGES_REQUESTED** [review]

**Comment by thangnguyen-novobi** [comment]
Changes made.

**Comment by thangnguyen-novobi** [comment]
Changes made.

**Pull request merged**